lib/vdsm/storage/managedvolume.py

```diff
@@ -113,17 +115,19 @@ def attach_volume(vol_id, connection_info):
                         "rbd, iscsi")
 
                 _add_udev_rule(vol_id, path)
+                _create_managedvolume_link_dir()
+                run_path = os.path.join(VOLUME_LINK_DIR, vol_id)
+                os.symlink(attachment['path'], run_path)
```

Comment:
I think we need _add_run_link() and _remove_run_link(), same as _add_udev_rule() and _remove_udev_rule().

We also need to handle cleaning up after creating run path failed - currently we leave leftover udev rule.